src/pages/library/Code.js

```diff
@@ -22,18 +22,19 @@ const styles = {
   }
 }
 
-const makeCard = method => {
+export const makeToolCard = (method, onClick) => {
```

Comment:
Rather than a function with positional args, now that this is shared it might be clearer as a component, e.g.
```
export const ToolCard = ({ method, onClick }) => {
```